api/src/controllers/contact.ts

```diff
@@ -45,3 +47,46 @@ export const createContact = async (
     throw errorWithCode(message, 500);
   }
 };
+
+/**
+ * POST /requests
+ * @param {Express Request} req
+ * @param {Express Response} res
+ * @returns undefined
+ */
```

Comment:
I like this inclusion, however I have a feeling it is a copy and paste from Patricks app. Could be benefitial to create a ticket to update the rest of the API to match.